src/common/pf_ppm.c

```diff
@@ -678,10 +681,13 @@ int pf_ppm_set_iocs (
          {
             LOG_ERROR (
                PF_PPM_LOG,
-               "PPM(%d): iocs_len %u expected length %u\n",
+               "PPM(%d): You give IOCS with size %u, but PLC expects size %u "
```

Comment:
Instead maybe:
"Incorrect IOCS length. Given size %u,  but PLC expects..."
<error message> <error reason>

I think we should avoid using  "you" in log messages.

Applies for all updated log messages.

Reply:
Done